src/rendering/renderers/autoDetectRenderer.ts

```diff
@@ -42,7 +42,7 @@ export async function autoDetectRenderer(options: Partial<AutoDetectOptions>): P
     let RendererClass: new () => Renderer;
 
     await autoDetectEnvironment(
-        options.manageImports ?? false,
+        options.manageImports ?? true,
```

Comment:
is this a correct change?

Reply:
it is

i realised i changed it to false by accident in a previous commit